lms/tasks/email_digests.py

```diff
@@ -40,35 +40,57 @@ def send_instructor_email_digest_tasks(batch_size):
 
     with app.request_context() as request:  # pylint:disable=no-member
         with request.tm:
-            h_userids = request.db.scalars(
-                select(User.h_userid)
-                .select_from(User)
-                .distinct()
-                .join(ApplicationInstance)
-                .join(AssignmentMembership)
-                .join(LTIRole)
-                .where(
-                    ApplicationInstance.settings["hypothesis"][
-                        "instructor_email_digests_enabled"
-                    ].astext
-                    == "true",
-                    LTIRole.type == "instructor",
-                )
-            ).all()
+            if not h_userids:
+                try:
+                    h_userids = request.db.scalars(
+                        select(User.h_userid)
+                        .select_from(User)
+                        .distinct()
+                        .join(ApplicationInstance)
+                        .join(AssignmentMembership)
+                        .join(LTIRole)
+                        .where(
+                            ApplicationInstance.settings["hypothesis"][
+                                "instructor_email_digests_enabled"
+                            ].astext
+                            == "true",
+                            LTIRole.type == "instructor",
+                        )
+                    ).all()
+                except Exception as exc:  # pylint:disable=broad-exception-caught
+                    LOG.exception(exc)
+                    report_exception(exc)
```

Comment:
I reckon this might do the sentry reporting twice.

Reply:
I've tested and it doesn't seem to